core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -28,7 +28,8 @@
                                short-summary="answerGroup | summarizeAnswerGroup : getCurrentInteractionId():getAnswerChoices():true"
                                is-active="$index === activeAnswerGroupIndex"
                                on-delete-fn="deleteAnswerGroup"
-                               outcome="answerGroup.outcome">
+                               outcome="answerGroup.outcome"
+                               extra-rules="answerGroup.rule_specs.length - 1">
```

Comment:
I think it would be better to pass num-rules="answerGroup.rule_specs.length" and have the logic happen inside the header.